### PR TITLE
chore: add project scaffolding

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

Project scaffolding including:
- GitHub
  - Actions
  - Issue template
  - PR Template
- Minimal go project structure
- Rock
- README
- License 
- CODEOWNERS
- CONTRIBUTING

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+#### Logs
+<!-- If applicable, add logs to help explain your problem. -->
+
+#### Environment
+
+- Software version(if relevant): <!-- e.g. 1.2 -->
+- Cloud Environment: <!-- e.g. GKE -->
+- Kubernetes version (output from `kubectl version --short`):
```
Comment (ghislainbourgeois) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: Those don't make as much sense for this project, I think. We could generalize to environment information.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Checklist:
+
+- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that validate the behaviour of the software
+- [ ] I validated that new and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have bumped the version of the library
```
Comment (ghislainbourgeois) [inline .github/pull_request_template.md:13]: Does bumping the library makes sense in this project?

File: rockcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+base: bare
+build-base: ubuntu@22.04
+version: '0.0.1'
+summary: A certificate management tool
+description: |
+  A certificate management tool.
+license: Apache-2.0
+platforms:
+  amd64:
+
```
Comment (ghislainbourgeois) [inline rockcraft.yaml:11]:
```suggestion
  amd64:
  arm64:

```

**Review by ghislainbourgeois — CHANGES_REQUESTED** [review]

File: rockcraft.yaml (hunk 3 above)
Comment (gruyaume) [reply rockcraft.yaml]: Done

File: .github/pull_request_template.md (hunk 2 above)
Comment (gruyaume) [reply .github/pull_request_template.md]: Nop, removed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (gruyaume) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed, changed

**Review by Kayra1 — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Review by saltiyazan — APPROVED** [review]

**Pull request merged**